karma.conf.js

```diff
@@ -72,7 +73,7 @@ module.exports = function (config) {
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
         // you can also add Chrome or other browsers too
-        browsers: ['PhantomJS'],
+        browsers: ['Chrome'],
```

Comment:
This particular change confuses me. Doesn't this mean headless mode will still run in Chrome?